common.mk

```diff
@@ -25,4 +48,13 @@ depsclean:
 	@rm -rf node_modules
 	@echo -e " $(OK) deps clean"
 
+# Shared recipes
+.PHONY: mocha-test
+
+mocha-test:
```

Comment:
I guess this is a target rather than `MOCHA_CMD` because you wanted to be able to resolve `TEST_REQUIRES` lazily? If instead we use a custom `test` target in `fluent-dom` (see my other suggestion), that problem would go away.

Reply:
It's a target 'cause I done this one first, and then the other ones didn't work as targets.

I'm pretty confident that this would work just as well as a `CMD`. Which is why I kinda looked at it with an angle when pushing.

I think I should just align that.